src/actions/guides/database/database_setup.cr

```diff
@@ -0,0 +1,39 @@
+class Guides::Database::DatabaseSetup < GuideAction
+  guide_route "/database/database-setup"
+
+  def self.title
+    "Database Setup"
+  end
+
+  def markdown
+    <<-MD
+    ## Configure
+
+    Maybe a mention of which version of postgres needs to be installed or min version?
```

Comment:
I think most recent versions work but I am not sure of an exact version. Prob v9, but depends on what you need more than Avram. Like if you need JSONB I think that is v9